### PR TITLE
CAL-1109: Adopt dependent filters collectLabelElements API changes

### PR DESCRIPTION
**Pull request by petrkolencik**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py [hunk 1]
```diff
@@ -552,7 +553,9 @@ def compute_valid_objects(self, workspace_id: str, ctx: ValidObjectsInputType) -
 
         return by_type
 
-    def get_label_elements(self, workspace_id: str, label_id: LabelElementsInputType) -> List[str]:
+    def get_label_elements(
+        self, workspace_id: str, label_id: LabelElementsInputType, depends_on: Optional[DependsOn] = None
+    ) -> List[str]:
```
Comment (hkad98) [inline gooddata-sdk/gooddata_sdk/catalog/workspace/content_service.py:558]: We do not want users to use classes from `gooddata_api_client` directly therefore, we create a wrapper class on top of it.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Review by hkad98 — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR424

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/244/ : FAILURE in 1m 12s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/394/ : SUCCESS in 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/398/ : SUCCESS in 9s

**Pull request merged**